tests/models/registry.py

```diff
@@ -353,6 +353,8 @@ def check_available_online(
     "Qwen2VLForConditionalGeneration": _HfExamplesInfo("Qwen/Qwen2-VL-2B-Instruct"),  # noqa: E501
     "Qwen2_5_VLForConditionalGeneration": _HfExamplesInfo("Qwen/Qwen2.5-VL-3B-Instruct",  # noqa: E501
                                                           min_transformers_version="4.49"),  # noqa: E501
+    "Qwen2_5OmniThinkerForConditionalGeneration": _HfExamplesInfo("Qwen/Qwen2.5-Omni-7B",  # noqa: E501
+                                                                  min_transformers_version="4.51"),  # noqa: E501
```

Comment:
```suggestion
                                                                  min_transformers_version="4.52"),  # noqa: E501
```

Actually we should update this to 4.52